.circleci/should_build_docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eo pipefail
+
+oldrev="origin/master"
+newrev="HEAD"
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PROJECT_DIR=$SCRIPT_DIR/..
+
+# Look for updated Dockerfiles in the PR
+cd "$PROJECT_DIR"
+COMMITS=$(git rev-list $oldrev..$newrev)
+DOCKER_FILES=$(
```

Comment:
nit: we do have auxiliary files as well. Would it make sense to test for a non-empty `git-diff-tree` on each `find libra/ -iname "*.Dockerfile" -execdir pwd \;`?

Reply:
Not sure I get your PoV on aux files.  Mind share it in another way? 

We want to build any docker file that has been updated/added in the PR (possibly multiple commits).